### PR TITLE
lvol: honor check_mode on thinpool

### LINKED ISSUE #2934: lvol with thinpool option ignores check mode
### Summary

When I run ansible in check mode lvol module ignores it and creates lv. This only happens if thinpool option is specified (thinpool already exists). 

### Issue Type

Bug Report

### Component Name

lvol

### Ansible Version

```console (paste below)
ansible==4.2.0
ansible-core==2.11.2
```

### Configuration

_No response_

### OS / Environment

target os: ubuntu 20.04 (focal)

### Steps to Reproduce

```yaml (paste below)
- hosts:
    - localhost
  tasks:
    - lvol:
        vg: vg0
        lv: test
        size: 1g
        thinpool: tpool
```

```console
ansible-playbook test.yml --check 
```

### Expected Results

I expect lvol to respect "check mode".

### Actual Results

First run:
```console (paste below)
PLAY [localhost] *******************************************************************************************************************************

TASK [Gathering Facts] ************************************************************************************************************************
ok: [localhost]

TASK [lvol] ***********************************************************************************************************************************
changed: [localhost]

PLAY RECAP ************************************************************************************************************************************
localhost                   : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```
Second run:
```console
PLAY [localhost] *******************************************************************************************************************************

TASK [Gathering Facts] ************************************************************************************************************************
ok: [localhost]

TASK [lvol] ***********************************************************************************************************************************
ok: [localhost]

PLAY RECAP ************************************************************************************************************************************
localhost                   : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by aminvakil**
##### SUMMARY
Fixes #2934 

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
plugins/modules/system/lvol.py

### CODE REVIEW

**Comment by aminvakil** [comment]
@pflorczyk Can you please confirm whether this PR fixes the issue?

You can also take a look at our [how to test PR a locally guide](https://github.com/ansible/community-docs/blob/main/test_pr_locally_guide.rst) on how to do so.

File: changelogs/fragments/2935-lvol-support_check_mode_thinpool.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - lvol - support check_mode on thinpool (https://github.com/ansible-collections/community.general/issues/2934).
```
Comment (felixfontein) [inline changelogs/fragments/2935-lvol-support_check_mode_thinpool.yml:3]: The module already supports `check_mode` - it just doesn't care about it for `thinpool`.
```suggestion
  - lvol - honor ``check_mode`` on thinpool (https://github.com/ansible-collections/community.general/issues/2934).
```

**Comment by pflorczyk** [comment]
With lvol.py from this pr i get:
```python
The full traceback is:
Traceback (most recent call last):
  File "/root/.ansible/tmp/ansible-tmp-1625483213.380403-13898-279438050385982/AnsiballZ_lvol.py", line 100, in <module>
    _ansiballz_main()
  File "/root/.ansible/tmp/ansible-tmp-1625483213.380403-13898-279438050385982/AnsiballZ_lvol.py", line 92, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/root/.ansible/tmp/ansible-tmp-1625483213.380403-13898-279438050385982/AnsiballZ_lvol.py", line 40, in invoke_module
    runpy.run_module(mod_name='ansible.modules.lvol', init_globals=dict(_module_fqn='ansible.modules.lvol', _modlib_path=modlib_path),
  File "/usr/lib/python3.8/runpy.py", line 207, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/usr/lib/python3.8/runpy.py", line 97, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/tmp/ansible_lvol_payload_3m19qxcu/ansible_lvol_payload.zip/ansible/modules/lvol.py", line 606, in <module>
  File "/tmp/ansible_lvol_payload_3m19qxcu/ansible_lvol_payload.zip/ansible/modules/lvol.py", line 474, in main
TypeError: not all arguments converted during string formatting
```

**Comment by aminvakil** [comment]
Just to be sure, code has been changed in https://github.com/ansible-collections/community.general/pull/2935/commits/148f01b392bb7e4975034c6341c253d28ff40ce9 to fix the same exact error you're facing.

@pflorczyk Are you sure you have checked out the latest file and not the one before you went to sleep? :)

**Comment by pflorczyk** [comment]
Sorry... I used: https://github.com/ansible-collections/community.general/pull/2935/commits/f6c925c39f315f4d8dd4264698c7668e40f6aceb. https://github.com/ansible-collections/community.general/commit/148f01b392bb7e4975034c6341c253d28ff40ce9 works perfectly. Thank you :)

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@aminvakil thanks for fixing this!
@pflorczyk thanks for testing!

**Pull request merged**